### PR TITLE
Fix/cancel modals

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

Old bridge issue
- in case of receive / xpub cancel, the `X` did nothing
- in case of review modal, it froze the app until user cancelled it on device

## Related Issue

Found during `react-hook-form` update PR

### CODE REVIEW

**Comment by tomasklim** [comment]
@trezor/qa  Should work as before for new bridge and webusb. Should not show X button in case of old bridge in confirmation stage of XPUB, Receive address, review transaction.

Please test also with remembered device.

File: packages/suite/src/components/suite/modals/confirm/ConfirmValueOnDevice.tsx [hunk 1]
```diff
@@ -72,6 +73,8 @@ export const ConfirmValueOnDevice = ({
 
     const showCopyButton = isConfirmed || !device.connected;
 
+    const isActionAbortable = useSelector(selectIsActionAbortable) || showCopyButton;
```
Comment (dahaca) [inline packages/suite/src/components/suite/modals/confirm/ConfirmValueOnDevice.tsx:76]: nit: selectors are usually at the start of the file :)

**Review by dahaca — APPROVED** [review]

**Comment by bosomt** [comment]
I will retest on web version as soon it get deployed to trezor.io url

**2.0.27 + dev app + model T**

- receive cancel OK    `Address verification error: Cancelled`
- xpub cancel OK       `Public key verification error: Cancelled`
- sign transaction cancel OK    `no notificatio no idea why`

**2.0.27 + dev app + remembered device**

- receive cancel -  OK `no notification`
- xpub cancel -  OK `no notification`

**2.0.33 + dev app + model T**

- receive cancel OK    `Address verification error: Cancelled`
- xpub cancel OK       `Public key verification error: Cancelled`
- sign transaction cancel OK    `no notificatio no idea why`x§

**2.0.33 + dev app + remembered device**

- receive cancel -  OK `no notification`
- xpub cancel -  OK `no notification`


**Info:**
 - Suite version: desktop 23.7.0 (818d910477c5961b351b8aa2087d8fa1e4a1a591)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/23.7.0 Chrome/112.0.5615.204 Electron/24.5.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.6.1 Universal (revision c3f6e8f19fd647c73b414322e287ba23cf8a0c7c)
 - Transport: NodeUsbTransport

**Comment by bosomt** [comment]
Retest with Bridge transport ;)


**2.0.27 + dev app + model T**

- receive `no X button`
- xpub `no X button`
- sign transaction `no X button`

**2.0.27 + dev app + remembered device**

- receive cancel -  OK `no notification`  `x button present`
- xpub cancel -  OK `no notification`.    `x button present`

**2.0.33 + dev app + model T**

- receive cancel OK    `Address verification error: Cancelled`
- xpub cancel OK       `Public key verification error: Cancelled`
- sign transaction cancel OK    `no notificatio no idea why`

**2.0.33 + dev app + remembered device**

- receive cancel -  OK `no notification`
- xpub cancel -  OK `no notification`




**Info:**
 - Suite version: desktop 23.7.0 (818d910477c5961b351b8aa2087d8fa1e4a1a591)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/23.7.0 Chrome/112.0.5615.204 Electron/24.5.1 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.6.1 Universal (revision c3f6e8f19fd647c73b414322e287ba23cf8a0c7c)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**